src/index.html

```diff
@@ -13,6 +13,52 @@
     />
```

Comment:
The attributes of the `<link>` element should be formatted with each attribute starting on a new line, with 2-space indentation relative to the tag.